### PR TITLE
Add customer shapes to fused SwiGLU kernel.

### PR DESCRIPTION
**Pull request by chengjunlu**
Add customer shapes to fused SwiGLU kernel.

### CODE REVIEW

**Review by whitneywhtsang — APPROVED** [review]

**Comment by chengjunlu** [comment]
@copilot Fix the CI failure by skip the accuracy test on BMG.

**Comment by whitneywhtsang** [comment]
> > @copilot Fix the CI failure by skip the accuracy test on BMG.
> 
> Fixed in commit `Skip accuracy test on BMG for fused SwiGLU benchmark`. Added `IS_BMG = 'B580' in DEVICE_NAME` and conditionally skip the `assert_close` accuracy check when running on BMG (B580) devices.

Why is it safe to not do verification?

File: benchmarks/triton_kernels_benchmark/fused_gemm_benchmark.py [hunk 1]
```diff
@@ -193,18 +196,26 @@ def fused_gemm_swiglu(x, w_g, w_fc, b_g, b_fc, M, N, K):
...
+        w_fc = torch.rand((K, N), device='xpu', dtype=torch.bfloat16)
+    b_g = torch.zeros((N, ), device='xpu', dtype=torch.bfloat16)
+    b_fc = torch.zeros((N, ), device='xpu', dtype=torch.bfloat16)
 
     if provider == 'triton':
         triton_fn = lambda: fused_gemm_swiglu(x, w_g, w_fc, b_g, b_fc, M, N, K)
         torch_fn = lambda: native_torch_fused_gemm(x, w_g, w_fc, b_g, b_fc)
-        rtol = 1e-2
-        benchmark_suite.assert_close(triton_fn, torch_fn, atol=1e-2, rtol=rtol, err_msg='triton to torch')
+        # Skip accuracy verification on BMG (B580): the kernel produces results outside
```
Comment (chengjunlu) [inline benchmarks/triton_kernels_benchmark/fused_gemm_benchmark.py]: @copilot No. It is because the reference path would hit OOM on BMG.

Comment (chengjunlu) [reply benchmarks/triton_kernels_benchmark/fused_gemm_benchmark.py]: @copilot Please just skip the accuracy test on BMG for the new added shapes.

**Comment by chengjunlu** [comment]
@copilot Please fix CI issue.

**Review by whitneywhtsang — APPROVED** [review]
